src/git.rs

```diff
@@ -0,0 +1,26 @@
+use git2::{Error, Repository, RepositoryInitOptions};
+use indicatif::ProgressBar;
+use std::path::Path;
+
+pub fn init_repository(
+    pb: &ProgressBar,
+    dest: impl AsRef<Path>,
+    description: Option<String>,
+) -> Result<(), Error> {
+    pb.set_message("Initializing local Git repository...");
+    let mut opts = RepositoryInitOptions::new();
+    opts.description(
+        description
+            .unwrap_or("Aqora competition".to_string())
+            .as_str(),
+    )
+    .no_reinit(true)
+    .initial_head("main");
```

Comment:
you'd have to check if this gets automatically putlled from the users .gitconfig. If it does, I wouldn't want to mess with their local configuration

Reply:
Ahh yep it's overriding user .gitconfig  conf /: I've removed it!